knative-operator/pkg/controller/knativeeventing/knativeeventing_controller.go

```diff
@@ -48,6 +49,9 @@ func newReconciler(mgr manager.Manager) reconcile.Reconciler {
 	if ns, required := os.LookupEnv(requiredNsEnvName); required {
 		client.Create(context.Background(), &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{
 			Name: ns,
+			Labels: map[string]string{
```

Comment:
Do we cover the kafka namespace?

Reply:
kafka namespace is deployed via serverless-opertor? Sorry I cannot find it.

Reply:
We use it in the Makefile at least check here:
https://github.com/openshift-knative/serverless-operator/blob/c76313f15a3a612a1a38b812dcfa8f688fdf00c6/hack/lib/strimzi.bash#L7